target/logwatch/maillog.conf

```diff
@@ -0,0 +1,32 @@
+##########################################################################
+# $Id$
+##########################################################################
+
+########################################################
+# This was written and is maintained by:
+#    Kenneth Porter <shiva@well.com>
+#
+# Please send all comments, suggestions, bug reports,
+#    etc, to shiva@well.com.
+########################################################
+
+# What actual file?  Defaults to LogPath if not absolute path....
+LogFile = syslog
+LogFile = mail/mail.log
+LogFile = mail/mail.log.1
+
+
+# If the archives are searched, here is one or more line
+# (optionally containing wildcards) that tell where they are...
+#If you use a "-" in naming add that as well -mgt
+Archive = syslog.*
+Archive = mail/mail.log.*.gz
+
+
+# Expand the repeats (actually just removes them now)
+*ExpandRepeats
+
+# Keep only the lines in the proper date range...
+*ApplyStdDate
```

Comment:
Is the '*' a typo?

Reply:
These come directly from the `/usr/share/logwatch/default.conf/logfiles/maillog.conf` file

To create the `maillog.conf` file that is in my PR is a modified version of the original.

Reply:
I understand. I would remove the comments on the top, because THIS file isn't maintained Kenneth 😉 

I guess ``/usr/share/logwatch/default.conf/logfiles/maillog.conf`` is read first and extended/overwritten by the local changes made in ``/etc/logwatch/conf/logfiles/maillog.conf``. That said, are the four lines at the end necessary?

Reply:
Oops.. yeah - I should have removed the comment at the top.

I've now reviewed the doc on logwatch http://www.stellarcore.net/logwatch/tabs/docs/HOWTO-Customize-LogWatch.html - and better understand the way to customize it.

Very good review - thank you.

Reply:
I've tested it and it seems to work 👍 The daily logwatch mail went from ~60KB to 170KB

Reply:
FWIW - I've been running with this change as well for days now.

It is also possible to test this by shelling into the container

```
docker exec -it mailserver bash
logwatch --range Yesterday
```

If you have an existing mail server container that has been running for more than 1 day (because until you have a logrotate happen, the problem doesn't manifest itself) - then you can test things being broken.

Then by adding the `/etc/logwatch/conf/logfiles/maillog.conf` file - testing again, will result in a complete logwatch showing email that is in the `/var/log/mail/mail.log.1` file.

Reply:
> If you have an existing mail server container that has been running for more than 1 day

fyi: The (rotated) logs are kept during container restarts (if you use the default `docker-compose.yml`)

Reply:
Agree - a new installation with no rotations (ever) would not have a `/var/log/mail/mail.log.1` file. 

I should have been more precise that you need to have data in a /var/log/mail/mail.log.1 file to test in a sane way.